.github/workflows/ci.yml

```diff
@@ -55,16 +55,19 @@ jobs:
         run: nix-shell --pure ghc.nix/shell.nix --command "hadrian/build -j --flavour=quickest test --test-root-dirs=testsuite/tests/programs"
 
       - name: Run nix develop - Boot and Configure
-        run: nix develop -Lv ghc.nix# -c bash -c "./boot && configure_ghc"
+        run: nix develop -Lv --fallback ghc.nix# -c bash -c "./boot && configure_ghc"
```

Comment:
Actually, I don't understand why we would need a fallback here. Can't we just restart the pipeline?

Reply:
The issue is, if a binary cache answers with a `4*` code, the build will fail, however, instead of failing, you can just start building whatever derivation you requested from the binary cache, yourself. This should really be the default setting and I'm currently adding it everywhere to avoid unnecessary failures. If you wish, I can just get rid of it, again.

Reply:
Ah, okay, I see.